### PR TITLE
Small cosmetic fixes for Memote.jl

### PR DESCRIPTION
**Pull request by stelmo**

### CODE REVIEW

**Review by exaexa — DISMISSED** [review]
LGTM

File: src/memote/checks/Annotation.jl [hunk 1]
```diff
@@ -19,31 +19,31 @@ is_annotated_gene(model, gid) = !isempty(gene_annotations(model, gid))
 """
 $(TYPEDSIGNATURES)
 
-Helper function to find all the annotations that are missing for a component in a model.
+Helper function to find all the annotations for a component in a model.
 """
-function _find_unannotated_components(
+function _find_annotated_components(
```
Comment (exaexa) [inline src/memote/checks/Annotation.jl:24]: why underscore? (this is not being exported right?)

File: src/memote/checks/Metabolites.jl [hunk 2]
```diff
@@ -66,28 +62,34 @@ end
...
-Helper function to find orphan or deadend metabolites. Specify `consumed=true`
-to consider orphan metabolites or `false` to consider deadend metabolites. Set
-`complete_medium=true` to open all boundary reactions to simulate a complete
-medium.
+Helper function to find orphan or deadend metabolites. Specify
+`only_consumed=true` to consider orphan metabolites or `false` to consider
+deadend metabolites.
 """
-function _find_orphan_or_deadend_metabolites(model::MetabolicModel; consumed = true)
+function _find_orphan_or_deadend_metabolites(model::MetabolicModel; only_consumed = true)
```
Comment (exaexa) [inline src/memote/checks/Metabolites.jl:69]: underscore?

File: src/memote/checks/Metabolites.jl [hunk 3]
```diff
@@ -96,20 +98,19 @@ end
 $(TYPEDSIGNATURES)
 
 Find all metabolites that can only (excludes reversible reactions) be consumed
-in the `model` by inspecting the stoichiometric matrix.
+in the `model` by inspecting the stoichiometric matrix and reaction bounds.
 """
 find_orphan_metabolites(model::MetabolicModel) =
-    _find_orphan_or_deadend_metabolites(model, consumed = true)
+    _find_orphan_or_deadend_metabolites(model, only_consumed = true)
```
Comment (exaexa) [inline src/memote/checks/Metabolites.jl:104]: `_`

File: src/memote/config.jl [hunk 4]
```diff
@@ -340,7 +341,7 @@ end
 network_config = NetworkConfig(
     condition_number = 1e9,
     cycle_tol = 1e-3,
-    blocked_tol = 1e-3,
+    blocked_tol = 1e-8,
```
Comment (exaexa) [inline src/memote/config.jl:344]: this is a bit too strict no? (with float32 solvers it's just going to fail everytime)

**Review by exaexa — APPROVED** [review]
lgtm; please have a look at the comments tho

File: src/memote/checks/Annotation.jl (hunk 1 above)
Comment (stelmo) [reply src/memote/checks/Annotation.jl:24]: the leading underscore was pre-existing, I can remove it

File: src/memote/checks/Metabolites.jl (hunk 2 above)
Comment (stelmo) [reply src/memote/checks/Metabolites.jl:69]: easier to read

File: src/memote/config.jl (hunk 4 above)
Comment (stelmo) [reply src/memote/config.jl:344]: It misses fluxes in iML1515 if it is that low... need to be strict :/

File: src/memote/checks/Annotation.jl (hunk 1 above)
Comment (stelmo) [reply src/memote/checks/Annotation.jl:24]: yeah it is not exported

Comment (exaexa) [reply src/memote/checks/Annotation.jl:24]: yeah my point is that now with better modularization the underscores are unnecessary. Feel free to remove later as we pass along this

File: src/memote/config.jl (hunk 4 above)
Comment (exaexa) [reply src/memote/config.jl:344]: :sob:

**Review by exaexa — APPROVED** [review]
cool

**Pull request merged**